src/rpc/net.cpp

```diff
@@ -1019,6 +1019,32 @@ static RPCHelpMan addpeeraddress()
     };
 }
 
+static RPCHelpMan setmnthreadactive()
+{
+    return RPCHelpMan{"setmnthreadactive",
+        "\nDisable/enable automatic masternode connections thread activity.\n",
+        {
+            {"state", RPCArg::Type::BOOL, RPCArg::Optional::NO, "true to enable the thread, false to disable"},
+        },
+        RPCResult{RPCResult::Type::BOOL, "", "The value that was passed in"},
+        RPCExamples{""},
+        [&](const RPCHelpMan& self, const JSONRPCRequest& request) -> UniValue
+{
+
+    if (Params().NetworkIDString() != CBaseChainParams::REGTEST) {
+        throw std::runtime_error("setmnthreadactive is for regression testing (-regtest mode) only.");
```

Comment:
should not it be JSONRPCError?

Reply:
For whatever reason all such messages are thrown this way 🤷‍♂️ 
https://github.com/search?q=repo%3Abitcoin%2Fbitcoin+%22%28-regtest+mode%29%22&type=code